Automattic/jetpack
# Whatsapp Block: Fix Guyana country code metadata

Fixes https://github.com/Automattic/wp-calypso/issues/47469

#### Changes proposed in this Pull Request:

* Updates Guyana's country code from +595 to +592

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* With your jetpack site, navigate to either the post or page editor
* Insert the whatsapp button block into the editor
<img width="859" alt="Screen Shot 2020-11-18 at 5 17 52 PM" src="https://user-images.githubusercontent.com/5414230/99608146-2bca3b00-29c2-11eb-9ec1-584fd19e9e4e.png">
* In block settings under the whatsapp button settings panel, click on the Phone Number dropdown
<img width="1059" alt="Screen Shot 2020-11-18 at 5 20 34 PM" src="https://user-images.githubusercontent.com/5414230/99608321-7c419880-29c2-11eb-833e-c410ae38b1f9.png">

* Scroll down the +592 and verify that the Guyana country code is present
* Select +592 as the country code
* Enter in a fictional number (Ex. 2255291)
* Publish the page or post
* Either preview the site or visit the frontend where the button is displayed
* Click on the button and verify that users are directed to message the correct phone number in whatsapp

#### Proposed changelog entry for your changes:

* Fix Guyana country code metadata for Whatsapp Button
